CONTRIBUTING.md

```diff
@@ -1,24 +1,62 @@
 # How to contribute to this project ?
 
-1. First you must choose an issue on [the project github](https://github.com/mdl29/scratchy). If you want to do something but no issue is created yet you must create one (try to tag it properly).
-2. Fork the project.
-3. Create a new branch whith the following name format on your fork : `resolves #ISSUEID Component: title for features/fix`.
+## **Choose a issue :**
```

Comment:
Titles are in bold by defaut, I think it's useless to use `**...**` in titles

Reply:
I mean it's more beautiful haha

Reply:
It doesn't change anything except adding characters in the source file ^^